egs2/chime4/asr1/local/data.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+SECONDS=0
+
+train_cmd="run.pl" # To fix 54 line shell check ^-- SC2154: train_cmd is referenced but not assigned.
+
```

Comment:
train_cmd= is meaningless because it's overridden by cmd.sh

Reply:
I deleted it now. I am worring about the CI check might fail.

Reply:
Oh, sorry I missed your comment, but don't worry. Now ci works well because it consider cmd.sh.

https://github.com/espnet/espnet/blob/24b07c73e0c3b98177d376cb7e0e5c936ed78a53/ci/test_shell.sh#L25

Reply:
Could you merge v.0.7.0 to reflect current CI?